modules/ingestor/src/service/format.rs

```diff
@@ -39,8 +41,14 @@ impl<'g> Format {
                 let loader = CveLoader::new(graph);
                 loader.load(source, reader, checksum).await
             }
-            Format::SPDX => todo!(),
-            Format::CycloneDX => todo!(),
+            Format::SPDX => {
+                let loader = SPDXLoader::new(graph);
```

Comment:
Just a nitpick  … It's `CveLoader`, but `SDPXLoader` and `CycloneDXLoader`. I would prefer to go for plain camel case. And I think Rust does too: https://rust-lang.github.io/rust-clippy/rust-1.78.0/index.html#/acron